### ISSUE TITLE
Namespace repairing generates xmlns definitions for xml: prefix (which is implicit)

### ISSUE THREAD

**Issue by mvysny** [FasterXML/jackson-dataformat-xml#395]
Saving this class:
```kotlin
data class StdTitle(@field:JacksonXmlProperty(isAttribute = true, namespace = "http://www.w3.org/XML/1998/namespace", localName = "lang") var lang: String = "")
```

will produce the following XML when namespace repairing is true:
```
<StdTitle xmlns:wstxns1="http://www.w3.org/XML/1998/namespace" wstxns1:lang="en"/>
```
However, the xml: prefix and the `http://www.w3.org/XML/1998/namespace` namespace is implicitly defined on all XMLs and there is no need to generate xmlns for that.

**Comment by cowtowncoder** [comment]
Interesting. I can reproduce this -- I am surprised Woodstox does try to output this as it shouldn't (as per XML specification). Will need to see what gives.

**Comment by cowtowncoder** [comment]
Fix will be included in 2.10.5 (which will take a while to release) and 2.11.1 (probably released earlier, in 1-2 months)
